grafana-plugin/src/containers/ColumnsSelector/ColumnsSelector.tsx

```diff
@@ -0,0 +1,273 @@
+import React, { useMemo, useRef } from 'react';
+
+import {
+  DndContext,
+  closestCenter,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+  DragEndEvent,
+} from '@dnd-kit/core';
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy,
+  useSortable,
+} from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+import { Button, Checkbox, Icon, IconButton, LoadingPlaceholder, Tooltip, useStyles2 } from '@grafana/ui';
+import { isEqual } from 'lodash-es';
+import { observer } from 'mobx-react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+
+import Text from 'components/Text/Text';
+import { WithPermissionControlTooltip } from 'containers/WithPermissionControl/WithPermissionControlTooltip';
+import { AGColumn, AGColumnType } from 'models/alertgroup/alertgroup.types';
+import { ActionKey } from 'models/loader/action-keys';
+import { LoaderStore } from 'models/loader/loader';
+import { useStore } from 'state/useStore';
+import { openErrorNotification } from 'utils';
+import { UserActions } from 'utils/authorization';
+import { WrapAutoLoadingState } from 'utils/decorators';
+
+import { getColumnsSelectorStyles } from './ColumnsSelector.styles';
+
+const TRANSITION_MS = 500;
+
+interface ColumnRowProps {
+  column: AGColumn;
+  onItemChange: (id: number | string) => void;
+  onColumnRemoval: (column: AGColumn) => void;
+}
+
+const ColumnRow: React.FC<ColumnRowProps> = ({ column, onItemChange, onColumnRemoval }) => {
+  const dnd = useSortable({ id: column.id });
+
+  const styles = useStyles2(getColumnsSelectorStyles);
+
+  const { attributes, listeners, setNodeRef, transform, transition } = dnd;
+  const columnElRef = useRef<HTMLDivElement>(undefined);
+
+  const style: React.CSSProperties = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
+
+  return (
+    <div ref={setNodeRef} style={{ ...style }} className={styles.columnRow}>
+      <div className={styles.columnItem} ref={columnElRef}>
+        <span className={styles.columnName}>{column.name}</span>
+
+        {column.type === AGColumnType.LABEL && (
+          <Tooltip content="Label Column">
+            <Icon aria-label="Label" name="tag-alt" className={styles.labelIcon} />
```

Comment:
if I'm not wrong removing this class and related !importants change nothing and the icon looks the same:
<img width="285" alt="image" src="https://github.com/grafana/oncall/assets/12073649/d765be80-2751-45e4-9ed1-f4e378092070">